corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py

```diff
@@ -226,6 +229,20 @@ def process_output(case_output, depth=0):
 
 
 def get_case_hierarchy(case, type_info):
+
+    def get_all_reverse_indices(case):
+        """
+        Return a list of child and extension type reverse indices for the given
+        case
+        """
+        children_indices = case.reverse_indices
```

Comment:
I'm not sure, I figured that `case.reverse_indices` is probably used in many different places, so that change would likely effect many pages. Looks like it's also used in `CommCareCase.get_json()`, so a change would affect some fundamental things, like APIs. @proteusvacuum, you implemented extension cases right? Do you have any thoughts on the matter?